client/my-sites/email/inbox-management/mailbox-selection-list/progress-line-style.scss

```
@@ -0,0 +1,101 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.progress-line__container {
+	position: fixed;
+	top: 50%;
+	left: 50%;
+	transform: translate( -50%, -50% );
+
+	@include break-large {
+		transform: none;
+	}
+
+
+	.progress-line__status-text-container {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		margin-bottom: 10px;
+
+		@include break-large {
+			margin-bottom: 20px;
+		}
+
+		.progress-line__status-text {
+			font-family: SF Pro Display, SF UI Text, SFUIText-Regular, Helvetica Neue, HelveticaNeue, Helvetica, Arial, sans-serif;
+			font-weight: 400;
+			font-size: $font-title-small;
+			line-height: 22px;
+
+			@include break-large {
+				font-size: $font-title-medium;
+				line-height: 28px;
+			}
+		}
+	}
+
+	.progress-line__loading-container {
+		position: relative;
+		width: 280px;
+		margin: 0 auto;
+		border-radius: 2px;
+		box-sizing: border-box;
+
+		@include break-medium {
+			width: 335px;
+		}
+
+		@include break-large {
+			width: 400px;
+		}
+
+		@keyframes load {
+			0% {
+				width: 0;
+				background: #3c434a;
```

This should be the value of the `--studio-gray-70` variable, here and below.
```suggestion
				background: var( --studio-gray-70 );
```

---

Thanks. FYI I did search for this - in vain